src/app.ts

```diff
@@ -242,6 +243,11 @@ fedify.setObjectDispatcher(
     undoDispatcher,
 );
 
+fedify.setNodeInfoDispatcher(
+    '/.ghost/activitypub/nodeinfo/2.1',
```

Comment:
I don't fully understand this - so does `fedify` serve a document on `/.well-known/nodeinfo` that links to this URL?

Reply:
Yes.

Reply:
When you register a `nodeInfoDispatcher` it enables `/.well-known/nodeinfo` routing (automagic) which when visited returns:

```
{
    "links":
    [
        {
            "rel": "http://nodeinfo.diaspora.software/ns/schema/2.1",
            "href": "https://mikebook-pro.tail5da2a.ts.net/.ghost/activitypub/nodeinfo/2.1",
            "type": "application/json; profile=\"http://nodeinfo.diaspora.software/ns/schema/2.1#\""
        }
    ]
}
```

and `href` is the link to our `nodeinfo`